server/controllers/list.js

```diff
@@ -877,7 +877,7 @@ const addViewer = (req, resp) => {
       return User.findOne({ email: sanitize(email) }).exec();
     })
     .then(userData => {
-      if (!userData) {
+      if (!userData || userData.isActive === false) {
```

Comment:
same here

Reply:
I think that in this case, I have to compare to false because of users authenticated via Google - they don't have property isActive. For them, it will be `undefined` so this comparison will be true then. Or I can check: ` if(!user || (!idFromProvider && !user.isActive)`